lib/widgets/components/assignment_card.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+class AssignmentCard extends StatelessWidget {
+  final String title;
+  final String description;
+  final Widget status;
+
+  const AssignmentCard({
+    super.key,
+    required this.title,
+    required this.description,
+    required this.status,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Card(
+      margin: const EdgeInsets.all(8),
+      child: Padding(
+        padding: const EdgeInsets.all(16),
+        child: Column(
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                Align(
+                  alignment: Alignment.topLeft,
+                  child: Text(
+                    title,
```

Comment:
You didn't handle Overflow. Try to put large text as title and see what happens.